packages/mobile/src/fiatExchanges/saga.ts

```diff
@@ -121,52 +117,59 @@ function* bidaliPaymentRequest({
   }
 }
 
-export function* searchNewItemsForProviderTxs({ transactions }: NewTransactionsInFeedAction) {
+export function* fetchTxHashesToProviderMapping() {
+  const account = yield call(getAccount)
+  const channel = yield call(providerTxHashesChannel, account)
```

Comment:
This is a channel that is meant to be used to continuously read from Firebase. Here we just want to read once IIUC. Instead of using the channel that requires somewhat more boilerplate code I think we could use `readOnceFromFirebase(``registrations/${address}/txHashes``)` which is a normal async function and will hopefully make this function a little simpler.

Reply:
Good suggestion, thanks